.cirrus.yml

```diff
@@ -19,7 +19,7 @@ env:
     UBUNTU_NAME: "ubuntu-2204"
 
     # Google-cloud VM Images
-    IMAGE_SUFFIX: "c6300530360713216"
+    IMAGE_SUFFIX: "c20230207t150657z-f37f36u2204"
```

Comment:
@cevich this is where I need to update the image right?

Reply:
correct.